consensus/service/BUILD.md

```diff
@@ -97,8 +97,8 @@ Recommended SDK and package installation:
 (
 	. /etc/os-release
 
-	wget "https://download.01.org/intel-sgx/sgx-linux/2.18/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_sdk_2.18.100.3.bin"
-	wget "https://download.01.org/intel-sgx/sgx-linux/2.18/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_driver_2.11.054c9c4c.bin"
+	wget "https://download.01.org/intel-sgx/sgx-linux/2.19/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_sdk_2.19.100.3.bin"
+	wget "https://download.01.org/intel-sgx/sgx-linux/2.19/distro/ubuntu${VERSION_ID}-server/sgx_linux_x64_driver_2.11.54c9c4c.bin"
```

Comment:
Note the new URL without the "0"
https://download.01.org/intel-sgx/sgx-linux/2.19/distro/ubuntu22.04-server
versus
https://download.01.org/intel-sgx/sgx-linux/2.18/distro/ubuntu22.04-server